deriving-aeson.cabal

```diff
@@ -23,7 +23,7 @@ library
   exposed-modules:
     Deriving.Aeson
     Deriving.Aeson.Stock
-  build-depends:       base >= 4.12 && <5, aeson >= 1.3 && <1.5
+  build-depends:       base >= 4.12 && <5, aeson >= 1.4.7.0 && <1.5
```

Comment:
This version constraint change may be controversial. The `rejectUnknownFields` was added in that release.

The alternative is to use conditional compilation and only add `RejectUnknownFields` behind a CCP guard.

Or reject this PR altogether ;)